pkg/admission/karydia/admission_pod.go

```diff
@@ -29,7 +29,7 @@ import (
 func (k *KarydiaAdmission) mutatePod(pod *corev1.Pod, ns *corev1.Namespace) *v1beta1.AdmissionResponse {
 	var patches patchOperations
 
-	seccompProfile, annotated := ns.ObjectMeta.Annotations["karydia.gardener.cloud/seccompProfile"]
+	seccompProfile, annotated := k.getSeccompProfileAnnotation(ns)
```

Comment:
We should think about renaming this, since it's not only about annotations but also implements the fallback to the karydiaConfig.